spec/spec_helper.rb

```diff
@@ -15,6 +15,10 @@
 CA_PEM = "#{SSL_CERTS_DIR}/ca.pem"
 CRL_PEM = "#{SSL_CERTS_DIR}/crl.pem"
 
+require 'timeout'
+require 'benchmark'
+require 'fileutils'
+require 'tmpdir'
 require 'mongo'
```

Comment:
We wouldn't need all these require's or the following helper methods. We can just test with the "$where" clause I put in the suggested test above.